todolist/server/src/main/scala/implicits.scala

```diff
@@ -45,33 +38,45 @@ sealed trait ServerImplicits extends CommonRuntime with RepositoriesImplicits {
     new TodoListRpcServiceHandler[IO]()
 
   implicit val todoItemRpcServiceHandler: TodoItemRpcService[IO] =
-    new TodoItemRpcServiceHandler[IO] {}
+    new TodoItemRpcServiceHandler[IO]()
 }
 
-sealed trait RepositoriesImplicits {
+sealed trait RepositoriesImplicits extends CommonRuntime {
+
+  implicit val timer: Timer[IO]     = IO.timer(EC)
+  implicit val cs: ContextShift[IO] = IO.contextShift(EC)
+  implicit val bl: Blocker          = Blocker.liftExecutionContext(EC)
 
   implicit val xa: HikariTransactor[IO] =
-    HikariTransactor[IO](new HikariDataSource(new HikariConfig(new Properties {
-      setProperty("driverClassName", "org.h2.Driver")
-      setProperty("jdbcUrl", "jdbc:h2:mem:todo")
-      setProperty("username", "sa")
-      setProperty("password", "")
-      setProperty("maximumPoolSize", "10")
-      setProperty("minimumIdle", "10")
-      setProperty("idleTimeout", "600000")
-      setProperty("connectionTimeout", "30000")
-      setProperty("connectionTestQuery", "SELECT 1")
-      setProperty("maxLifetime", "1800000")
-      setProperty("autoCommit", "true")
-    })))
+    HikariTransactor[IO](
+      new HikariDataSource(
+        new HikariConfig(
+          new Properties {
+            setProperty("driverClassName", "org.h2.Driver")
+            setProperty("jdbcUrl", "jdbc:h2:mem:todo")
+            setProperty("username", "sa")
+            setProperty("password", "")
+            setProperty("maximumPoolSize", "10")
+            setProperty("minimumIdle", "10")
+            setProperty("idleTimeout", "600000")
+            setProperty("connectionTimeout", "30000")
+            setProperty("connectionTestQuery", "SELECT 1")
+            setProperty("maxLifetime", "1800000")
+            setProperty("autoCommit", "true")
+          }
+        )
+      ),
+      EC,
+      bl
+    )
```

Comment:
Not necessary for this PR but I'd use the `newHikariTransactor` def which includes the default values and returns a `Resource[IO, HikariTransactor[IO]]`:
```suggestion
HikariTransactor.newHikariTransactor[IO](
  "org.h2.Driver",
  "jdbc:h2:mem:todo",
  "sa",
  "",
  EC,
  bl
)
```
This would require a small refactoring but IMHO it'd be better handle the transactor as a resource in the application.

Reply:
yeah I actually tried that initially (since it's what the doobie docs recommended) but ended up failing to instantiate the DB connection when running the app.  That said, that was before I ended up getting the hikari transactor working at all, so now that I know it's working, I can try using the `newHikariTransactor` def.